config/targets.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-env node */
+
+module.exports = {
+  browsers: [
```

Comment:
for the purposes of learning it might be easier to not include this to avoid introducing a potentially confusing concept?